### PR TITLE
Chart: cacheTTL

### PR DESCRIPTION
**Pull request by pbalogh-sa**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    |yes
| API breaks?     | no
| Deprecations?   | no
| Related tickets | #72 
| License         | Apache 2.0

### What's in this PR?
set cache TTL in values

### CODE REVIEW

**Review by bonifaido — APPROVED** [review]

File: charts/anchore-policy-validator/README.md [hunk 1]
```diff
@@ -55,3 +55,4 @@ The following tables lists configurable parameters of the anchore-policy-validat
 |namespaceSelector                    |webHookConfig additional namespaceSelector behaviour|{}                                 |
 |objectSelector                       |webHookConfig additional objectSelector behaviour|{}                                    |
 |createPolicies                       |create some predifined policy bundles        |false                                     |
+|config.app.cacheTTL                  |setup cacehTTL                               |6h                                        |
```
Comment (orymate) [inline charts/anchore-policy-validator/README.md:58]:
```suggestion
|config.app.cacheTTL                  |duration between resubmissions of an image|6h                                        |
```

File: cmd/config.go [hunk 2]
```diff
@@ -19,7 +19,7 @@ import (
 	"os"
 	"strings"
 
-	"github.com/banzaicloud/anchore-image-validator/internal/app"
+	"github.com/banzaicloud/anchore-	image-validator/internal/app"
```
Comment (orymate) [inline cmd/config.go:22]: ?

Comment (pbalogh-sa) [reply cmd/config.go]: !

Comment (pbalogh-sa) [reply cmd/config.go]: thanks

**Pull request merged**